pages/windows/driverquery.md

```diff
@@ -0,0 +1,31 @@
+# driverquery
+
+> Display information about installed device drivers.
+
+- Display a list of all installed device drivers:
+
+`driverquery`
+
+- Display a list of drivers in the specified format:
+
+`driverquery /fo {{table|list|csv}}`
+
+- Display a list of drivers with a column for if they are signed:
```

Comment:
s/with a column for if they are signed/with a column to indicate if they are signed/.